aws/amazon-ecs-cli-v2
# fix: app logs scale up issue

fixes #826.

## Manual Test
```$ ecs-preview app logs --follow```
```
...
714beb4 10.0.1.247 - - [09/Apr/2020 18:06:59] "GET / HTTP/1.1" 200 -
714beb4 10.0.0.79 - - [09/Apr/2020 18:06:59] "GET / HTTP/1.1" 200 -
714beb4 10.0.1.247 - - [09/Apr/2020 18:07:09] "GET / HTTP/1.1" 200 -
714beb4 10.0.0.79 - - [09/Apr/2020 18:07:09] "GET / HTTP/1.1" 200 -
714beb4 10.0.1.247 - - [09/Apr/2020 18:07:19] "GET / HTTP/1.1" 200 -
714beb4 10.0.0.79 - - [09/Apr/2020 18:07:19] "GET / HTTP/1.1" 200 -
714beb4 10.0.1.247 - - [09/Apr/2020 18:07:29] "GET / HTTP/1.1" 200 -
714beb4 10.0.0.79 - - [09/Apr/2020 18:07:30] "GET / HTTP/1.1" 200 -
8b2270b  * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
8b2270b  * Restarting with stat
8b2270b  * Debugger is active!
8b2270b  * Debugger PIN: 155-519-084
714beb4 10.0.1.247 - - [09/Apr/2020 18:07:39] "GET / HTTP/1.1" 200 -
714beb4 10.0.0.79 - - [09/Apr/2020 18:07:40] "GET / HTTP/1.1" 200 -
714beb4 10.0.1.247 - - [09/Apr/2020 18:07:49] "GET / HTTP/1.1" 200 -
714beb4 10.0.0.79 - - [09/Apr/2020 18:07:50] "GET / HTTP/1.1" 200 -
714beb4 10.0.1.247 - - [09/Apr/2020 18:07:59] "GET / HTTP/1.1" 200 -
714beb4 10.0.0.79 - - [09/Apr/2020 18:08:00] "GET / HTTP/1.1" 200 -
714beb4 10.0.1.247 - - [09/Apr/2020 18:08:09] "GET / HTTP/1.1" 200 -
714beb4 10.0.0.79 - - [09/Apr/2020 18:08:10] "GET / HTTP/1.1" 200 -
714beb4 10.0.1.247 - - [09/Apr/2020 18:08:19] "GET / HTTP/1.1" 200 -
714beb4 10.0.0.79 - - [09/Apr/2020 18:08:20] "GET / HTTP/1.1" 200 -
8b2270b 10.0.0.79 - - [09/Apr/2020 18:08:20] "GET / HTTP/1.1" 200 -
8b2270b 10.0.1.247 - - [09/Apr/2020 18:08:20] "GET / HTTP/1.1" 200 -
...
```

As we can see, when I scaled up the service, `app logs` now is able to detect task `8b2270b` was created and started showing logs for it.




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
